usagov/vote-gov-nvrf-app
# VOTE-2726 Add error message if data fails to load

https://cm-jira.usa.gov/browse/VOTE-2726

**Testing instructions:**
1. Change the lang in index.html to ko and make sure the error message displays
2. Change the lang to `es` and delete the PDF file `public/es/nvrf/data/Federal_Voter_Registration_es.pdf` and make sure the error message displays
